src/metrics/nom.rs

```diff
@@ -240,6 +240,25 @@ mod tests {
         );
     }
 
+    #[test]
+    fn c_nom() {
```

Comment:
git is not smart enough to notice it was just moved, please do the move in a separate commit or PR, otherwise this change will mess up the blame info.